hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -416,6 +464,53 @@ private void ingestExternalFile() throws RocksDBException {
         }
     }
 
+    private RocksDB createSnapshotRocksDB(String snapshotPath)
```

Comment:
add `opentRocksDB(config, dataPath, walPath)` method, and let RocksDBStdSessions()/reloadRocksDB()/hadrlinkSnapshot() call it.